src/components/ProductList/ProductList.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+
+  <ul class="items-container"> 
+    <ProductListItem v-for="item in items" 
+    :id="item.id" 
+    :key="item.id"
+    :title="item.title" 
+    :url="item.url"
+    :price="item.price"
+     @delete-item="openDeleteWindow"
+     @editing-item="openEditWindow"
```

Comment:
`edit-item` would probably be more correct, especially considering the delete event name. 😃